docs/mtest/mtest/RotationMatrix.md

```diff
@@ -44,9 +44,9 @@ automatically.
 
 ~~~~{.cpp}
 // standard way
-@RotationMatrix {{0,1,0},
-                 {1,0,0},
-                 {0,0,1}};
+@RotationMatrix {{cos,-sin, 0},
```

Comment:
maybe add a comment about the angle used to compute the cosine and sinus ?
This angle is the angle between x and the first orthotropic direction, isn't it ?

Reply:
I've added precision to the angle definition, for the standard and <Direction> options.